.babelrc

```diff
@@ -0,0 +1,18 @@
+{
+  "plugins": [
+    "transform-react-jsx",
+  ],
+  "presets": [
+    "es2015",
+    "react",
+    "stage-0"
+  ],
```

Comment:
Instead of these, could you use `@babel/preset-env` and the stitchfix browserslist? This will let us compile for our specific browsers and nothing more.

Reply:
Sure! Sorry for the newbie ❓  ... does this mean I should remove the `"presets"` node here, and instead include `@babel/preset-env` and the stitchfix browserlist in `package.json`?

Reply:
Yep, exactly! The modern version of babelrc is babel.config.js too. You can check KIUI or kufak-fe-infra for examples of babel configurations.

In fact, you could probably add kufak-fe-infra to this package, and use our shared babel config. Check out Ben's PR in Knit for an example of how it's done.

Reply:
Since this is technically an open source project, I was trying to limit adding Kufak/Stitch Fix specific config that wouldn't be available to devs outside Stitch Fix. So I will try and emulate what you all did in kufak-fe-infra but scope it to this project.

Reply:
Ahhhhh! That makes a lot of sense!

If you wanna be real modern and OSS cool, you can add a `modules` entry in package.json that points to the source code entry point. That way anyone can grab the source code and compile using their own babel settings.

Reply:
Ok, not sure what you mean by `modules` ... do you mean `main`, as in [here](https://github.com/stitchfix/log_weasel/pull/28/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2R7)?

Reply:
Well it turns out `modules`, although used and spoken about pretty often, is not yet an accepted proposal by npm. Info here:
https://levelup.gitconnected.com/code-splitting-for-libraries-bundling-for-npm-with-rollup-1-0-2522c7437697#9f6f
and here:
https://github.com/rollup/rollup/wiki/pkg.module
and the original proposal here:
https://github.com/dherman/defense-of-dot-js/blob/master/proposal.md#typical-usage

The benefit would be: 
1. if a user is only calling the logweasel ID generator in their webpack apps
2. by adding the source code directly to the webpack dependency tree
3. they will ensure that the source code will be transpiled, minified, and code split to the best of their app's ability. If any necessary polyfills, deps, or transpilation helpers are shared by this package and the rest of the webpack bundle, they'll be de-duped and code split in the most efficient way possible.
4. If you were able to guarantee that 1 is always true, you could discard the webpack process in this app, and the assignment to window, and let this script be pulled in and referenced as a source-code only dependency. This is what CAR and Knit do.

Detriments:
1. If you need to call this outside of a webpack bundle, it aint gonna work :P

Reply:
Gotcha, so I paired with @benmvp this morning and he helped bring configs up to date. I think we should be good now.